SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1414,6 +1415,31 @@ void USpatialReceiver::ResolveObjectReferences(FRepLayout& RepLayout, UObject* R
 
 				ObjectProperty->SetObjectPropertyValue(Data + AbsOffset, SinglePropObject);
 			}
+			else if (ObjectReferences.bFastArrayProp)
```

Comment:
We're we just not handling unresolved refs for fast arrays?

Reply:
Previously fast arrays were handled like standard arrays, so this custom logic wasn't necessary.